skale/contracts/dkg.py

```diff
@@ -20,6 +20,21 @@
 from skale.contracts import BaseContract, transaction_method
 from skale.transactions.tools import retry_tx
 from skale.utils.constants import GAS
+from skale.utils.helper import split_public_key
+
+
+class KeyShare():
```

Comment:
I think it's better to remove braces in the end. It doesn't do anything except two extra symbols.

Reply:
+